token-swap/program/src/error.rs

```diff
@@ -102,6 +102,12 @@ pub enum SwapError {
     /// The operation cannot be performed on the given curve
     #[error("The operation cannot be performed on the given curve")]
     UnsupportedCurveOperation,
+    /// Incorrect swap guardian was provided with the operation.
+    #[error("The provided swap guardian is incorrect")]
+    WrongSwapGuardian,
```

Comment:
This should probably be called `IncorrectSwapGuardian` to match to wording in the other errors.

Reply:
Done.